src/StatsdClient/BufferBuilder.cs

```diff
@@ -0,0 +1,45 @@
+using System.Text;
+
+namespace StatsdClient
+{
+    /// <summary>
+    /// Append string values to a fixed size bytes buffer.
+    /// </summary>
+    class BufferBuilder
```

Comment:
Declare accessibility of classes explicitly, e.g., use `public` if that is what is intended.

Reply:
I know it is very common to always add accessibility to classes and methods but I would like to know if there is any rational behind this? (In the same way, I do not create default constructor 
 for example)

Reply:
It doesn't affect the compiled code but it is done for team consistency and for readability.

Reply:
Thanks for the explanation!